linked_list/linked_list.py

```diff
@@ -23,13 +28,67 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+    # sets value of node to new_node
+    new_node = Node(value)
```

Comment:
The comment is slightly off....all you're doing here is initializing a new node called new_node with some value.